docs/customize.md

```diff
@@ -61,7 +61,7 @@ This Python script can be used to generate a sample `~/.visidatarc`, with all of
 ~~~
 from visidata import options
 
-for optname, val in sorted(options().items()):
+for optname, val in sorted(dict(options).items()):
```

Comment:
This is a clever way of doing it, but I think it should be `sorted(options.get_all().items())`.

Reply:
Yup, your way is better. I was just quickly hacking to make it work.